compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -38,15 +39,60 @@ import qualified DA.Daml.LF.Ast as LF
 import DA.Daml.Preprocessor.Generics
 import SdkVersion
 
+data Config = Config
+    { configPackages :: MS.Map UnitId LF.Package
+    , configGetUnitId :: LF.PackageRef -> UnitId
+    , configStablePackages :: Set LF.PackageId
+    , configDependencyPackages :: Set LF.PackageId
```

Comment:
Might be nice to add a comment here explaining the relationship between `configPackages`, `configStablePackages` and `configDependencyPackages`. I guess `configPackages` includes stable packages, dependencies and data-dependencies?

Reply:
Improved the docs here. I also switched gave `envQualify` a more explicit name and switched it (`True -> False`) since it was confusing to me.